File: webservices/CORE/Rol.php
<?php


class Rol
{
    public $idrol;
    public $nombre;
    public $estado;


    public function getIdrol()
    {
        return $this->idrol;
    }

    /**
     * @param mixed $idrol
     */
    public function setIdrol($idrol)
    {
        $this->idrol = $idrol;
    }

    /**
     * @return mixed
     */
    public function getNombre()
    {
        return $this->nombre;
    }

    /**
     * @param mixed $nombre
     */
    public function setNombre($nombre)
    {
        $this->nombre = $nombre;
    }

    /**
     * @return mixed
     */
    public function getEstado()
    {
        return $this->estado;
    }

    /**
     * @param mixed $estado
     */
    public function setEstado($estado)
    {
        $this->estado = $estado;
    }



}

File: Facturador/app/src/main/java/com/example/facturador/Activity/Model/Rol.java
package com.example.facturador.Activity.Model;

import java.io.Serializable;

public class Rol implements Serializable {
    private int idrol;
    private String nombre;
    private String estado;
    private int id_base;

    public int getId_base() {
        return id_base;
    }

    public void setId_base(int id_base) {
        this.id_base = id_base;
    }

    public Rol(int idrol, String nombre, String estado) {
        this.idrol = idrol;
        this.nombre = nombre;
        this.estado = estado;
    }

    public Rol() {

    }

    public int getIdrol() {
        return idrol;
    }

    public void setIdrol(int idrol) {
        this.idrol = idrol;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getEstado() {
        return estado;
    }

    public void setEstado(String estado) {
        this.estado = estado;
    }
}
